src/seedu/addressbook/Main.java

```diff
@@ -109,6 +110,13 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
+        } catch (StorageOperationException e){
+			//try loading file again, else empty file is recreated
+        	try {
+				storage.load();
```

Comment:
- In exception handling first there needs to be a place where you are throwing an exception when the file is not found. Hine: save() in StorageFile

- Then you need to catch and handle it. Deciding to handle it in executeCommand is a good idea. 

- Next proper handling would be showing a warning to the user and continuing with the program after recovery. So throwing another RunTimeException is not a good way to handle it. 

- Spend a little more time trying to understand this concept.